src/core/api/index.js

```diff
@@ -1,4 +1,5 @@
 /* global fetch */
+/* eslint-disable arrow-body-style */
```

Comment:
Oh maybe I was unclear, I actually would prefer we used explicit `return` *always*; I dislike the implicit return syntax and invariable refactoring of the code once it becomes multi-line/statement.

Anywho, this is better, I was just nitpicking about disabling a rule whole-file over line, really 😄